### DISCUSSION TITLE
Tailwind Play linked on daisyUI homepage can't find module daisyui

### DISCUSSION THREAD

**Discussion by MeikyuuTrader** [saadeghi/daisyui#1058] [General]
[Homepage Tailwind Play](https://play.tailwindcss.com/gTlq3iys55) from [this](https://daisyui.com/tailwindplay) gives a config error, can't find module daisyui. Can just use the codepen to try, so just a notice of that error. 

Haven't touched anything, just opened up tailwind play and let it load, then it gives that error. Was something supposed to be added or compiled first?

**Comment by rhscjohn-dev** [comment]
I did not get a Config error when I tried both of your links.  I got good result on both links.

**Reply by saadeghi** [reply]
Only has errors on Safari and Firefox. Works fine on Chrome and Edge 🤔

**Comment by saadeghi** [comment]
This seems to be a problem from Tailwind Play because it works fine on Chrome and Edge but throws error on Safari and Firefox.  
I submitted an issue on Tailwind Play repo: https://github.com/tailwindlabs/play.tailwindcss.com/issues/161

**Comment by saadeghi** [comment]
It is fixed now 🎉
